spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Volume }
+  let(:level) { an_instance_of(Fixnum) }
+
+  describe '.up' do
+    it 'calls up.scpt' do
+      expect(Volume).to receive(:execute_script).with('volume/up.scpt', level)
+      Volume.up
+    end
+  end
+
+  describe '.down' do
+    it 'calls down.scpt' do
+      expect(Volume).to receive(:execute_script).with('volume/down.scpt', level)
+      Volume.down
+    end
+  end
+
+  describe '.mute' do
+    it 'calls mute.scpt' do
+      expect(Volume).to receive(:execute_script).with('volume/mute.scpt')
+      Volume.mute
+    end
+  end
+
+  describe '.unmute' do
+    it 'calls unmute.scpt' do
+      expect(Volume).to receive(:execute_script).with('volume/unmute.scpt')
+      Volume.unmute
+    end
+  end
+
+  describe '.value' do
+    subject { volume.value }
+    context 'when iTunes returns volume level' do
+      before { expect(Volume).to receive(:execute_script).with('volume/value.scpt').and_return(volume.value) }
```

Comment:
sorry but this is what I get when I do ```and_return(level)```:
![schermata 2015-11-04 alle 09 31 09](https://cloud.githubusercontent.com/assets/1325491/10933201/fe75b5ae-82d6-11e5-822b-64a1868f05ca.png)
```value``` method should return a string indicating the volume level, I don't know how to fix CI :(